example.js

```diff
@@ -0,0 +1,35 @@
+// Require the necessary discord.js classes
+const { Client, Intents } = require('discord.js');
+const { token } = require('./config.json');
+const roverify = require("roverify");
+
+// Create a new client instance
+const client = new Client({ intents: 513 });
```

Comment:
Using other intents here may be a good idea; as the [guide](https://discordjs.guide/popular-topics/intents.html) shows, you can use strings here instead.